nat-lab/tests/telio.py

```diff
@@ -166,7 +166,6 @@ class AdapterType(Enum):
 
 class Runtime:
     _output_notifier: OutputNotifier
-    _peer_info: Dict[str, PeerInfo]
```

Comment:
simillar logic should apply to `_derp_state` , should be removed and used last event from `_derp_state_events`

Reply:
missed that, fixed!

Reply:
however, shouldn't we make `_derp_state_events/_peer_state_events` atomic? At the moment we have different threads writing/reading them without synchronization

Reply:
Idk if we need it for now, but you could create Jira issue and add relation to LLT-4054

Reply:
After some thinking, I've concluded that's not worth to make them atomic. We have only one thread writing them and even if race conditions happen it's not critical as the readers (wait_for_events) will just read them again with no side effects (notify state/event functions)